core-s3/src/test/scala/io/aiven/guardian/kafka/s3/MinioContainer.scala

```diff
@@ -10,13 +10,13 @@ class MinioContainer(accessKey: String, secretKey: String)
       "minio/minio",
       exposedPorts = List(9000),
       waitStrategy = Some(Wait.forHttp("/minio/health/ready").forPort(9000).withStartupTimeout(Duration.ofSeconds(10))),
-      command = List("server", "data"),
+      command = List("server", "/data"),
```

Comment:
Not sure why `"data"` without the "/" works but best to use what minion officially documents at https://hub.docker.com/r/minio/minio/